dragnet/content_extraction_model.py

```diff
@@ -124,5 +122,67 @@ def plot(blocks, content_mask):
 
         fig.show()
 
+
+class ContentCommentsExtractionModel(ContentExtractionModel):
+    '''
+    Run two models: a content only and a content + comments model
+    on a document and return the output of both
+    '''
+    def __init__(self, blockifier, features,
+        content_model, content_comments_model, threshold=0.5):
+
+        self._blockifier = blockifier
+        self._features = features
+        self._content_model = content_model
+        self._content_comments_model = content_comments_model
+        self._threshold = threshold
+
+        # check the features
+        self._nfeatures = sum(ele.nfeatures for ele in self._features)
+        for f in self._features:
+            assert callable(f)
```

Comment:
I'm not a big fan of assertions in production code, but it's not a religious issue for me.